eth_tester/backends/pyevm/main.py

```diff
@@ -164,9 +175,16 @@ class MainnetTesterNoProofChain(MiningChain):
         def validate_seal(cls, block):
             pass
 
-    genesis_params = get_default_genesis_params()
-    account_keys = get_default_account_keys()
-    genesis_state = generate_genesis_state(account_keys)
+    if genesis_params is None:
+        genesis_params = get_default_genesis_params()
+
+    if genesis_state:
+        accounts = len(genesis_state)
```

Comment:
If you like, `num_accounts` could be a little clearer as a name.

Reply:
:heavy_check_mark: